app/views/general/404.phtml

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Can you share a screenshot? Did we test this across screen sizes / different browsers?

Reply:
Chen approved it already